src/univers/version_range.py

```diff
@@ -446,7 +480,7 @@ def from_native(cls, string):
         return cls(constraints=constraints)
 
 
-def split_req(string, comparators, default=None, strip=""):
+def split_req(string, comparators, comparators_rear={}, default=None, strip=""):
```

Comment:
what is the definition of  `comparator_rear`?

Also, if you want to account for the Maven, GitLab, Snyk and NuGet range syntaxes, you likely want to have another function for this entirely.

Unless there are cases in the wild where the two notations -- intervals boundaries with `[]()` vs. logical `><=`) -- are mixed together in a single expression?  I would surprised that this ever happens, hence two functions would be best.

Do you have some example links to paste?

Reply:
Done!